front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/index.ts

```diff
@@ -87,12 +87,24 @@ async function handler(
   auth: Authenticator,
   space: SpaceResource
 ): Promise<void> {
+  if (!space.canList(auth)) {
```

Comment:
can you add the space logic we have in other endpoints?

Reply:
app.space.sId === space.sId :pray:

Reply:
we can't here, it's a list , apps are not passed in url by fetched from the space

Reply:
true that

Reply:
this logic is already performed via the withResourceFromRoute wrapper; IMO we shouldn't duplicate the check